internal/install/recipe_installer_guided.go

```diff
@@ -153,12 +154,17 @@ func (i *RecipeInstaller) installLogging(ctx context.Context, m *types.Discovery
 		"matches": acceptedLogMatches,
 	}).Debug("matches accepted")
 
-	// The struct to approximate the logging configuration file of the Infra Agent.
-	type loggingConfig struct {
-		Logs []types.LogMatch `yaml:"logs"`
+	discoveredLogFiles := []string{}
+	for _, logMatch := range acceptedLogMatches {
+		discoveredLogFiles = append(discoveredLogFiles, logMatch.File)
 	}
 
-	r.AddVar("DISCOVERED_LOG_FILES", loggingConfig{Logs: acceptedLogMatches})
+	discoveredLogFilesString := strings.Join(discoveredLogFiles, ",")
```

Comment:
It's tricky to get this deployed with the library changes.
Could we add a new data element maybe named "NR_DISCOVERED_LOG_FILES" in addition of the existing "DISCOVERED_LOG_FILES", and then in a subsequent release remove the old element "DISCOVERED_LOG_FILES" (after all recipes have been updated to use the new data element).

Reply:
Oh right, thanks for catching that.

Reply:
Updated 😎